engine/admin/logging/awslogs.md

```diff
@@ -77,6 +77,115 @@ $ docker run --log-driver=awslogs \
 > **Note**:
 > Your AWS IAM policy must include the `logs:CreateLogGroup` permission before you attempt to use `awslogs-create-group`.
 
+### awslogs-multiline-pattern
+
+The `awslogs-multiline-pattern` specifies a regular expression pattern for identifying the start of a log message.  A log message is made of a line that matches the pattern and any following lines that don't match the pattern.  
+
+You cannot configure this option in conjunction with the `awslogs-datetime-format` option.
+
+> **Note**:
+> Multiline logging performs regular expression parsing and matching of all log messages, which may have a negative impact on logging performance.
+
+For example, to process the following log stream where new log messages start with the pattern `INFO`:
+
+```
+INFO A message was logged
+INFO Another multiline message was logged
+     Some random message
+INFO Another message was logged
+```
+
+The regular expression `^INFO` can be used:
+
+```bash
+$ docker run --log-driver=awslogs \
+             --log-opt awslogs-region=us-east-1 \
+             --log-opt awslogs-group=myLogGroup \
+             --log-opt awslogs-multiline-pattern='^INFO' \
+             ...
+```
+
+Which will generate the following CloudWatch log events:
+
+```
+# First event
+INFO A message was logged
+
+# Second event
+INFO Another multiline message was logged
+     Some random message
+
+# Third event
+INFO Another message was logged
+```
+
+### awslogs-datetime-format
+
+The `awslogs-datetime-format` specifies an [strftime expression pattern](http://strftime.org) for identifying the start of a log message.  A log message is made of a line that matches the pattern and any following lines that don't match the pattern.  
+
+You cannot configure this option in conjunction with the `awslogs-multiline-pattern` option.
+
+> **Note**:
+> Multiline logging performs regular expression parsing and matching of all log messages, which may have a negative impact on logging performance.
+
+For example, to process the following log stream where new log messages start with the following timestamps:
+
+```
+[May 01, 2017 19:00:01] A message was logged
+[May 01, 2017 19:00:04] Another multiline message was logged
+Some random message
+with some random words
+[May 01, 2017 19:01:32] Another message was logged
+```
+
+An strftime expression of `[%b %d, %Y %H:%M:%S]` can be used:
+
+```bash
+$ docker run --log-driver=awslogs \
+             --log-opt awslogs-region=us-east-1 \
+             --log-opt awslogs-group=myLogGroup \
+             --log-opt awslogs-datetime-format='[%b %d, %Y %H:%M:%S]' \
+             ...
+```
+
+Which will generate the following CloudWatch log events:
+
+```
+# First event
```

Comment:
Can you change "First event", "Second event", and "Third event" to use "entry" instead of event?